test/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-6.xml

```diff
@@ -911,6 +911,29 @@
 		<output type="failure" caseSensitive="no" regex="no">corrupt</output>
 	</test>
 	
+	<test id="Test 207: Create a non-persistent cache" timeout="600" runPath=".">
+		<command>$JAVA_EXE$ -Xtrace:print=j9prt.700 $currentMode$,nonpersistent -version</command>
+		<output type="success" caseSensitive="yes" regex="yes" javaUtilPattern="yes">(java|openjdk) version</output>
```

Comment:
Shouldn't this check for the tracepoint output as well as a success condition?

Reply:
The tracepoint output (ControlFileFDWithWriteLock (Info: could not chown file.)) is a failure condition. chown() failure is something that should not happen.

Reply:
Rather than adding yet another test, could we just modify an existing test? i.e. Test 26 in test/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-1.xml creates a nonpersistent cache.

Reply:
There is another one here test/cmdLineTests/shareClassTests/SCCMLTests/testSCCMLSnapshot.xml
Test 2s

Reply:
testSCCMLSnapshot.xml does not run on windows.

Reply:
right, I forgot about that. lgtm then.